src/content/en/fundamentals/accessibility/a11y-for-teams.md

```diff
@@ -0,0 +1,341 @@
+and `span`, are not opted into the tab order. This means if you use a `div` to
+create an interactive control, you'll need to do some additional work to make
+it keyboard accessible.
+
+Two options are:
+
+- Give the control a `tabindex="0"`. This will at least make it focusable,
+  though you'll likely need to do additional work to add support for keypresses.
+- Where possible, consider using a `button` instead of a `div` or `span` for any
+button-like control. The native `button` element is very easy to style and gets
+keyboard support for free.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="CZGqnp06DnI"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+### Managing focus
+
+When you change the content of the page, it's important to direct the user's
+attention by moving focus. A classic example of when this technique is useful is
+opening a modal dialog. If a user relying on a keyboard presses a button to open
+a dialog and their focus _is not_ moved into the dialog element, then their only
+course of action is to tab through the entire site until they eventually can
+interact with the new control. By moving focus into the new content as soon as
+it appears, you can improve the efficiency of these users’ experiences.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="JS68faEUduk"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+Also note that there may be times where you want to _prevent_ focus from
+entering an area. In these situations the newly proposed [HTML `inert`
+property](https://github.com/wicg/inert) may come in handy. `inert` says that an
+element and all of its descendants should not be keyboard focusable or be part
+of the accessibility tree. This can be very useful when you want to turn off a
+section of the page to effectively trap keyboard focus in just one area. Using
+the dialog example from before, you may want to mark the rest of the page as
+`inert` so the user can only take action on the dialog while it is displaying.
+
+### Keyboard support for interactive elements
+
+If you're building a custom control like a carousel or dropdown, then you'll
+need to do some additional work to add in keyboard support. The [ARIA Authoring
+Practices Guide](https://w3.org/TR/wai-aria-practices-1.1) is a useful resource
+which identifies various UI patterns and the kinds of keyboard actions they are
+expected to support.
+
+![An excerpt from the ARIA Authoring Practices guide explaining how to build a radio group.](imgs/aria-authoring.jpg)
+
+To learn more about adding in keyboard support to an element, take a look at the [roving tabindex](https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex#managing_focus_in_components) section in Google’s Accessibility Fundamentals docs.
```

Comment:
Change 'adding in' to 'adding'. In general try not to use prepositions where they're not needed.